subsys/spm/Kconfig

```diff
@@ -11,6 +11,14 @@ menuconfig SPM
 
 if SPM
 
+config PARTITION_MANAGER_RESERVED_SPACE_SPM
```

Comment:
This symbol needs to have an SPM_ prefix, as well, as every other symbol in this file.

Reply:
This is PARTITION_MANAGER specific, the ´PARTITION_MANAGER_RESERVED_SPACE_´ is used by the `partition_manager.py` script to deduce the configured sizes of the various partitions.

Reply:
Why are some Partition Manager macros prefixed with just `PM_` e.g. `PM_CFG_SPM_ADDRESS` while other with `PARTITION_MANAGER`, is there a difference?

Reply:
As we discussed in the mcuboot pr for this, the define which needs to be in Kconfig looks very out of place with just a `PM` prefix. This is why the full `PARTITION_MANAGER` prefix is used.

It should be the only place that `PM_` is not used.